scapy/utils.py

```diff
@@ -228,118 +265,125 @@ def hexdump(x, dump=False):
                 s += "%02X " % orb(x[i + j])
             else:
                 s += "   "
-        s += " %s\n" % sane_color(x[i:i + 16])
+        s += " %s\n" % sane(x[i:i + 16], color=True)
         i += 16
     # remove trailing \n
     s = s[:-1] if s.endswith("\n") else s
     if dump:
         return s
     else:
         print(s)
+        return None
 
 
 @conf.commands.register
-def linehexdump(x, onlyasc=0, onlyhex=0, dump=False):
+def linehexdump(p, onlyasc=0, onlyhex=0, dump=False):
+    # type: (Union[Packet, AnyStr], int, int, bool) -> Optional[str]
     """Build an equivalent view of hexdump() on a single line
 
     Note that setting both onlyasc and onlyhex to 1 results in a empty output
 
-    :param x: a Packet
+    :param p: a Packet
     :param onlyasc: 1 to display only the ascii view
     :param onlyhex: 1 to display only the hexadecimal view
     :param dump: print the view if False
     :return: a String only when dump=True
     """
     s = ""
-    s = hexstr(x, onlyasc=onlyasc, onlyhex=onlyhex, color=not dump)
+    s = hexstr(p, onlyasc=onlyasc, onlyhex=onlyhex, color=not dump)
     if dump:
         return s
     else:
         print(s)
+        return None
 
 
 @conf.commands.register
-def chexdump(x, dump=False):
+def chexdump(p, dump=False):
+    # type: (Union[Packet, AnyStr], bool) -> Optional[str]
     """Build a per byte hexadecimal representation
 
     Example:
         >>> chexdump(IP())
         0x45, 0x00, 0x00, 0x14, 0x00, 0x01, 0x00, 0x00, 0x40, 0x00, 0x7c, 0xe7, 0x7f, 0x00, 0x00, 0x01, 0x7f, 0x00, 0x00, 0x01  # noqa: E501
 
-    :param x: a Packet
+    :param p: a Packet
     :param dump: print the view if False
     :return: a String only if dump=True
     """
-    x = bytes_encode(x)
+    x = bytes_encode(p)
     s = ", ".join("%#04x" % orb(x) for x in x)
     if dump:
         return s
     else:
         print(s)
+        return None
 
 
 @conf.commands.register
-def hexstr(x, onlyasc=0, onlyhex=0, color=False):
+def hexstr(p, onlyasc=0, onlyhex=0, color=False):
+    # type: (Union[Packet, AnyStr], int, int, bool) -> str
     """Build a fancy tcpdump like hex from bytes."""
-    x = bytes_encode(x)
-    _sane_func = sane_color if color else sane
+    x = bytes_encode(p)
     s = []
     if not onlyasc:
         s.append(" ".join("%02X" % orb(b) for b in x))
     if not onlyhex:
-        s.append(_sane_func(x))
+        s.append(sane(x, color=color))
     return "  ".join(s)
 
 
 def repr_hex(s):
+    # type: (bytes) -> str
     """ Convert provided bitstring to a simple string of hex digits """
     return "".join("%02x" % orb(x) for x in s)
 
 
 @conf.commands.register
-def hexdiff(x, y, autojunk=False):
+def hexdiff(a, b, autojunk=False):
```

Comment:
IMHO, the renaming of x/y to a/b is not necessary. More meaningful names might be better such as chunk1/chunk2, binary1/binary2 ...

Reply:
This renaming was actually only done because `x` and `y` were used to carry very different types 😄
But you're right since we're at it, we can rename them.